.circleci/config.yml

```diff
@@ -30,6 +30,7 @@ jobs:
             pip install -r requirements-ci.txt
             pip install -r requirements-cidocs.txt
             pip install pyinstaller
+            make frontend_deps
```

Comment:
We need to build guanlecoja-ui and buildbot_data in order to be able to do `pip install -e www/base`, because setup.py assumes these packages have already been built. I agree that the situation can be improved - we can extract only the `yarn build` part to a separate target.

Reply:
Should be fixed now.